scm/driver/harness/pr.go

```diff
@@ -234,9 +229,21 @@ func convertPullRequest(src *pr) *scm.PullRequest {
 			ID:    src.Author.UID,
 			Email: src.Author.Email,
 		},
-		Fork:   "fork",
-		Ref:    fmt.Sprintf("refs/pullreq/%d/head", src.Number),
-		Closed: src.State == "closed",
+		Head: scm.Reference{
+			Name: src.SourceBranch,
+			Path: scm.ExpandRef(src.SourceBranch, "refs/heads"),
+			Sha:  src.SourceSHA,
+		},
+		Base: scm.Reference{
+			Name: src.TargetBranch,
+			Path: scm.ExpandRef(src.TargetBranch, "refs/heads"),
+			Sha:  src.MergeSHA,
+		},
+		Fork:    "fork",
```

Comment:
should we set fork to anything - I'd assume it should be empty, no?

Reply:
i think empty string can cause some issue for upper grpc consumer

Reply:
oh okay we can keep it - seems odd though